### DISCUSSION TITLE
How can we use the emulator with postgres dialect support?

### DISCUSSION THREAD

**Discussion by ufaruq** [GoogleCloudPlatform/cloud-spanner-emulator#139] [Q&A]
I am unable to find any instructions for running the emulator with postgres dialect support. I am basically trying to run the emulator with the pgadapter, similar to my production environment in my unit tests.

**Comment by skuruppu** [comment]
@ufaruq, unfortunately we haven't officially released the postgresql dialect support in the emulator yet which is why we haven't given any instruction yet on how to run it. It is coming very soon though.

Please keep an eye out on https://github.com/GoogleCloudPlatform/cloud-spanner-emulator/issues/69 as I will update the issue once we've cut a release with PG support.

**Comment by skuruppu** [comment]
PostgreSQL dialect support was released in [v1.5.12](https://github.com/GoogleCloudPlatform/cloud-spanner-emulator/releases/tag/v1.5.12).
